Userland/Libraries/LibC/fcntl.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2021, the SerenityOS developers.
```

Comment:
I encourage you to use your own name (and e-mail) in copyright headers. It's fun to take some credit for things you worked hard on :^)

Reply:
Is a real name required? The main reason I attribute my copyright to the SerenityOS developers is to stay anonymous. If `Copyright (c) 2021, sin-ack <sin-ack@protonmail.com>` is acceptable, I can add that instead.

Reply:
Yes, that's okay. :)